javascript/index.js

```diff
@@ -119,10 +151,8 @@ function decode(paymentRequest){
             TAGPARSER = tlv_invoice
             break;
     default:
-	// FIXME: Mention prefix
-        throw new Error('Not a proper lightning payment request')  
+        throw new Error(prefix.toString() + ' is not a proper lightning prefix')  
     }
-    // console.log(type)
     for (let i=0;i<encodedData.length;i++){
         words[words.length]=ALPHABET_MAP[encodedData.charAt(i)]
     }
```

Comment:
Here, not above, is where you should check if a character is in the map?  We're iterating anyway.